### ISSUE TITLE
One version for users and one for administrator

### ISSUE THREAD

**Issue by alekbless** [saleem-hadad/larecipe#63]
Hello! 

Thank you for this wonderful package! 

I'am currently not using the package for versioning the docs, but to show different documentation based on the user. 
Say I have docs version "User Manual" and "Configuration". Can I show "User Manual" to one group of user based on a condition, and "Configuration" to another group? 

The key element here is that I've got documentation that is not needed for all users, the configuration part should only be shown to administrator.

**Comment by alekbless** [comment]
Would it be better to keep one version and rather give/reject access to specific pages?

**Comment by saleem-hadad** [comment]
Thank you @alekbless for your feedback, yes it's a nice feature to have 👍 
can you submit a PR to add this?

**Comment by alberto1el** [comment]
I sent a pull request that you can take a look, I did configurable guard per published version.
@alekbless 
https://github.com/saleem-hadad/larecipe/pull/67

**Comment by saleem-hadad** [comment]
I've added this feature now and it will be available in the next version 1.2.6 soon 💃 (6b7ad72)[https://github.com/saleem-hadad/larecipe/commit/6b7ad723e749fd707a9cda95fc0f251c1d23640c]
Now you can check which version and you take the appropriate action based on the user role:
```php
Gate::define('viewLarecipe', function($user, $documentation) {
    // check here using $documentation->version
    return true;
});
```
